.rubocop.yml

```diff
@@ -1,4 +1,6 @@
 Documentation:
   Enabled: false
   LineLength:
-    Max: 99
\ No newline at end of file
+    Max: 99
+  Style/FrozenStringLiteralComment:
+    Enabled: false
```

Comment:
is possible that Rubocop doesn't take these configs, indent to left after line 2

Reply:
Do I comment on lines 2 to 6?

Reply:
not, only move to left

Reply:
ahhh, why is that option within documentation

Reply:
this config is global shouldn’t  inside documentation